pkg/cmd/set_replication.go

```diff
@@ -113,8 +112,8 @@ func (o *FinalizeOptions) Bind(cmd *cobra.Command, v *viper.Viper) error {
 	v.AddConfigPath(".")
 	v.SetConfigType("yaml")
 	if err := v.ReadInConfig(); err != nil {
-		var nf *viper.ConfigFileNotFoundError
-		if !errors.As(err, &nf) {
+		//nolint:errorlint
+		if _, ok := err.(viper.ConfigFileNotFoundError); !ok {
```

Comment:
Maybe try using a non-pointer type for the error:
```go
    var nf viper.ConfigFileNotFoundError
```

See https://github.com/spf13/viper/issues/1139